SimDataFormats/TrackerDigiSimLink/interface/PixelSimHitAddExtraInfo.h

```diff
@@ -0,0 +1,41 @@
+#ifndef PixelSimHitAddExtraInfo_h
+#define PixelSimHitAddExtraInfo_h
+
+#include "DataFormats/GeometryVector/interface/LocalPoint.h"
+#include <vector>
+#include <cstdint>
+
+class PixelSimHitAddExtraInfo {
```

Comment:
Perhaps it would be better to call the class `PixelSimHitAddInfo` or `PixelSimHitExtraInfo`.

Reply:
ok, I can change the name, I prefer`PixelSimHitExtraInfo` among your 2 proposals.